packages/commonwealth/server/controllers/server_proposal_methods/get_proposal_votes.ts

```diff
@@ -44,7 +43,7 @@ export async function __getProposalVotes(
     );
     votes = votesArgs.map((vote) => formatCompoundProposalVote(vote));
   } else {
-    throw new ServerError(
+    throw new Error(
```

Comment:
This will be removed in #8326 but technically this should throw `ServerError` imported from `@hicommonwealth/core`.